src/system/obj/Dir.h

```diff
@@ -263,11 +266,15 @@ class ObjectDir : public virtual Hmx::Object {
     StringTable mStringTable; // 0x28
     FilePath mProxyFile; // 0x38
     bool mProxyOverride; // 0x44
+    /** "Can this proxy be inlined?" */
```

Comment:
Instead of formatting this as a question, do something like "If this proxy can be inlined."